csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest-docker-gpu.sh

```diff
@@ -13,44 +13,28 @@ CurrentOnnxRuntimeVersion=$4
 PackageName=${PACKAGENAME:-Microsoft.ML.OnnxRuntime.Gpu}
 RunTestCsharp=${RunTestCsharp:-true}
 RunTestNative=${RunTestNative:-true}
-#CUDA_VER=cuda10.0-cudnn7.3, cuda9.1-cudnn7.1, cuda10.0-cudnn7.3
-CUDA_VER=${5:-cuda10.0-cudnn7.3}
 
 PYTHON_VER=3.5
 IMAGE="ubuntu16.04-$CUDA_VER"
 OldDir=$(pwd)
 
 cd $SOURCE_ROOT/tools/ci_build/github/linux/docker
 
-DOCKER_FILE=Dockerfile.ubuntu_gpu_cuda9
-if [ $CUDA_VER = "cuda10.0-cudnn7.3" ]; then
-DOCKER_FILE=Dockerfile.ubuntu_gpu
-fi
+sudo --preserve-env docker build -t onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 --build-arg OS_VERSION=16.04 --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER}  -f Dockerfile.ubuntu_gpu .
```

Comment:
>--preserve-env [](start = 5, length = 14)

Was not env preserved before?

Reply:
I guess it's not. See the changes in #4063

Reply:
yep. `env` and `sudo env` may be quite different